src/style.css

```diff
@@ -71,51 +122,7 @@ body .popup .user span {
   margin-left: 30px;
 }
 
-footer {
-  display: box;
-  display: flexbox;
-  display: flex;
-  flex-wrap: wrap;
-  background-color: black;
-  color: #a4b1b8;
-  font-family: Volte, sans-serif;
-  -ms-flex-pack: distribute;
-  justify-content: space-around;
-  padding: 30px;
-  gap: 10px;
-}
-
-footer div {
-  -ms-flex-preferred-size: 300px;
-  flex-basis: 300px;
-}
-
-footer div h2 {
-  font-weight: bold;
-  font-size: 28px;
-  border-bottom: solid 2px #a4b1b8;
-  display: inline-block;
-}
-
-footer div p {
-  font-size: 20px;
-  color: white;
-}
-
-footer div a {
-  color: white;
-  display: block;
-  font-size: 20px;
-  text-decoration: none;
-  margin: 10px;
-}
-
-footer div:first-child {
-  text-align: center;
-  padding: 0 30px;
-}
-
-.comment p {
+body .popup .comment p {
```

Comment:
Good job so far @Behnam1369 :+1: I would suggest you to better stylize the content of the popup for a better user experience.